src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -254,7 +254,11 @@ class WalletStore extends BaseStore implements Wallet {
       this.notificationsStore.showMessage('Please change network to Holesky', 'error');
     } else {
       try {
-        changeCurrentNetwork(Number(networkId));
+        const chainId = networkId === NETWORKS.HOLESKY ? NETWORKS.MAINNET : networkId;
```

Comment:
Can you change the logic to automatically switch to mainnet if user connected to anything except mainnet?
You have is mainnet helper to bring boolean logic in here, and I propose to not stick with specific networks and just identify if it is not mainnet

Reply:
But I need to identify only if is not holesky, it can be mainnet or goerli but cannot be holesky